numpy/ma/core.py

```diff
@@ -5860,17 +5860,8 @@ def __getstate__(self):
 
         """
         cf = 'CF'[self.flags.fnc]
-        state = (1,
-                 self.shape,
-                 self.dtype,
-                 self.flags.fnc,
-                 self._data.tobytes(cf),
-                 # self._data.tolist(),
-                 getmaskarray(self).tobytes(cf),
-                 # getmaskarray(self).tolist(),
-                 self._fill_value,
-                 )
-        return state
+        data_state = super(MaskedArray, self).__reduce__()[2]
```

Comment:
I'm pretty comfortable with this, given that (1) the spec of `__reduce__` is mostly [prescribed by Python](https://docs.python.org/3/library/pickle.html#object.__reduce__
) and this MaskedArray is defined in NumPy itself, which means that if any ever changes how `ndarray.__reduce__` works our unit tests will catch the issue for MaskedArray.